src/component-systems/InstancingSystem.js

```diff
@@ -87,6 +94,7 @@ class InstancingSystem extends System {
         entity.object3D.add(instancedMesh);
     }
 
+// TODO - can i delete the below two items?
```

Comment:
@michaelthatsit - quick question for you

Reply:
if you're referring to updated and detached component, yeah you can delete them if you're not utilizing them.